INSTALL.rst

```diff
@@ -259,6 +262,62 @@ In order to build DFTB+ carry out the following steps:
+directory).
+
+Enabling optional code functionality
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
++-----------------+---------+------------------------------------------------+
+|cmake option     |Default  |Notes                                           |
++=================+=========+================================================+
+|-DWITH_ARPACK    |N        |Required for some types of excited state        |
+|                 |         |calculations (the Stratmann solver does not     |
+|                 |         |require this library).                          |
++-----------------+---------+------------------------------------------------+
+|-DWITH_CHIMES    |N        |Required for the CHiMES repulsive potentials to |
+|                 |         |be enabled.                                     |
++-----------------+---------+------------------------------------------------+
+|-DWITH_MBD       |N        |Required for many-body dispersion to be enabled.|
++-----------------+---------+------------------------------------------------+
+|-DWITH_PLUMED    |N        |Required for the PLUMED meta-dynamics library to|
+|                 |         |be available for MD calculations.               |
++-----------------+---------+------------------------------------------------+
+|-DWITH_POISSON   |N        |Required for Poisson solver (also enabled for   |
+|                 |         |transport builds).                              | 
++-----------------+---------+------------------------------------------------+
+|-DWITH_SDFTD3    |N        |Required for 'simple' DFT-D3 dispersion to be   |
+|                 |         |enabled.                                        |
++-----------------+---------+------------------------------------------------+
+|-DWITH_SOCKETS   |N        |Required for i-PI socket interfaces to be       |
+|                 |         |available.                                      |
++-----------------+---------+------------------------------------------------+
+|-DWITH_TBLITE    |N        |Required for xTB hamiltonians to be enabled.    |
++-----------------+---------+------------------------------------------------+
+|-DWITH_TRANSPORT |N        |Required for open-boundary calculations to be   |
+|                 |         |enabled.                                        |
++-----------------+---------+------------------------------------------------+
+
+Parallelism options
+^^^^^^^^^^^^^^^^^^^
+
++-----------------+--------+--------------------------------------------------+
+|Cmake option     | Default| Notes                                            |
++=================+========+==================================================+
+|-DWITH_OMP       | Y      |OpenMP parallelism enabled in the build.          |
++-----------------+--------+--------------------------------------------------+
+|-DWITH_MPI       | N      |MPI parallelism enanbled in the build.            |
```

Comment:
Need an extra white space for the table, but yes should be changed.
```suggestion
|-DWITH_MPI       | N      |MPI parallelism enabled in the build.             |
```

Reply:
Sorry :D